templates/control/2fa/disable.html

```diff
@@ -0,0 +1,29 @@
+$def with (username, error)
+$:{RENDER("common/stage_title.html", ["Disable 2FA", "Two-Factor Authentication", "/control/2fa/status"])}
+
+<div class="content form clear">
+  <form method="POST" class="form skinny clear" action="/control/2fa/disable">
+    $if error == "2fa":
+      <div id="signin_error"><strong>Whoops!</strong> The 2FA response token or recovery code you provided did not validate.</div>
+    $elif error == "verify":
+      <div id="signin_error"><strong>Hey!</strong> Did you want to disable 2FA? You didn't check the verification checkbox!</div>
+    <h3>Disable Two-Factor Authentication (2FA)</h3>
+    <p><strong>Username: $:{MARKDOWN_EXCERPT(username)}</strong></p>
```

Comment:
This is why I am using ``$:{MARKDOWN_EXCERPT(username)}`` versus using ``$:{MARKDOWN(username)}``. The excerpt version isn't giving me the additional carriage return in this instance, and frankly I am unsure why it is doing that.

![screenshot from 2017-03-12 19 51 58](https://cloud.githubusercontent.com/assets/19718911/23837425/7fc61c72-075e-11e7-8c71-907206e250f0.png)